Sources/StreamVideoSwiftUI/Utils/VideoRendererPool/VideoRendererPool.swift

```diff
@@ -47,6 +47,25 @@ final class VideoRendererPool: @unchecked Sendable {
     func releaseRenderer(_ renderer: VideoRenderer) {
         pool.release(renderer)
     }
+
+    /// Configures the global video renderer pool with the specified initial capacity.
+    /// This method is intended for internal use by StreamVideo.
+    ///
+    /// - Parameter initialCapacity: The number of video renderers to pre-create.
+    @MainActor
+    static func configure(initialCapacity: Int) async {
```

Comment:
This is what is happening on the reusePool initialisation. I don't see a reason for this method.